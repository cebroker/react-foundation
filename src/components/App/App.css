.App {
  text-align: center;
}

.App-brand {
  padding-top: 21px;
  padding-bottom: 20px;
  height: auto;
  float: none;
  display: block;
}

.nav {
	float: left;
}

.nav a {
	outline-offset: -5px;
}

.nav > li {
	display: inline-block;
}

.nav > li > a {
	padding-top: 24px;
	padding-bottom: 24px;
	color: rgba(255, 255, 255, 0.59);
	font-size: 13px;
}

.nav > li > a:hover,
.nav > li > a:focus {
	color: #fff;
	background: transparent;
}

.nav > li > a.dropdown-toggle:focus,
.nav > li > a.dropdown-toggle:active {
	outline: 5px auto -webkit-focus-ring-color;
	background-color: transparent;
}

.nav >li.dropdown.open > a.dropdown-toggle {
	background-color: transparent;
}

.nav > li.dropdown > .dropdown-menu {
	top: 63px;
}

.AppBar .search-form {
	display: flex;
	float: left;
	padding-top: 9px;
	padding-bottom: 9px;
	min-width: 306px;
	margin-top: 14px;
	margin-right: 15px;
	border-radius: 4px;
	background: rgba(255, 255, 255, 0.15);
	box-shadow: none;
}

.AppBar .search-form i {
	color: #fff;
	font-size: 20px;
	margin-right: 10px;
}

.appbar .red-indicator.active:after {
	border-color: #0a4b69;
}

.nav > li.upgrade-link a {
	color: #fff;
	font-weight: bold;
	letter-spacing: 0.5px;
	font-size: 11px;
	padding-top: 25px;
	padding-bottom: 25px;
}

.nav > li.avatar-dropdown {
	float: right;
}

.nav > li.avatar-dropdown > .dropdown-toggle > .avatar {
	line-height: 14px;
}

.nav > li.avatar-dropdown > a {
	padding-top: 20px;
	padding-bottom: 20px;
}

.nav > li.avatar-dropdown .dropdown-menu a > i {
	color: #b0b0b0;
	font-size: 16px;
	position: relative;
	top: 2px;
	margin-right: 9px;
}

.nav > li.avatar-dropdown .username {
	color: #4C5656;
	font-size: 13px;
	font-weight: 600;
	text-transform: none;
	margin: 0;
}

.nav > li.avatar-dropdown .subtitle {
	margin: 0;
	font-size: 12px;
	letter-spacing: 0.45px;
}

.nav > li.nav-item-icon > a {
	height: 66px;
	padding-top: 11px;
	padding-bottom: 10px;
}

.nav > li.nav-item-icon > a > .red-indicator {
	top: 9px;
}

.AppBar .search-form input {
	font-size: 13px;
	background-color: transparent;
	color: #fff;
}

.App-intro {
  font-size: large;
}

.ec-page-header {
	background: #0b5171;
	position: relative;
}

.ec-page-header .page-header-illustration {
	position: absolute;
	right: 0;
	bottom: 0;
}

.ec-page-header .page-header-nav {
	background: rgba(11, 81, 113, 0.8);
}

.page-header-nav.nav.nav-pills > li > a {
	color: rgba(255, 255, 255, 0.59);
	font-size: 14px;
}

.page-header-nav.nav.nav-pills > li.active > a {
	box-shadow: inset 0 -3px 0 #59C697;
	color: #f1f1f1;
	font-size: 14px;
}

.page-header-nav.nav.nav-pills > li > a > span > i {
	font-size: 19px;
	position: relative;
	top: 3px;
}

.dropdown-menu > li > a > i {
	font-size: 22px;
	margin-right: 17px;
	position: relative;
	top: 4px;
}

.search-form-clear-button,
.search-form-clear-button:before {
	height: 20px;
	width: 20px;
}