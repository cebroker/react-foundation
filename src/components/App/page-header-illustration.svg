<svg xmlns="http://www.w3.org/2000/svg" width="407" height="143" viewBox="0 0 407 143" xmlns:xlink="http://www.w3.org/1999/xlink">
  <defs>
    <polygon id="a" points="2.332 .022 .032 .022 .032 26.671 2.332 26.671 2.332 .022"/>
    <polygon id="c" points="17.814 16.495 17.814 .042 .02 .042 .02 16.495 17.814 16.495"/>
  </defs>
  <g fill="none" fill-rule="evenodd">
    <path fill="#FFFFFF" d="M348.892953 7.49464587C347.808257 4.31942645 344.384936 1.79371417 339.416634 2.34746922 332.824571-2.64971815 322.994468.754502753 321.56547 8.56264987 314.161526 12.6732935 318.013765 21.2621883 325.473877 21.2621883L349.839053 21.2621883C354.33467 21.2621883 358.133659 18.1110743 358.133659 14.3579945 358.133659 10.5218848 354.063314 7.04266925 348.892953 7.49464587M32.2057208 22.8270799C31.5534588 20.9177201 29.4949076 19.3989295 26.5073107 19.7319199 22.5432947 16.7269532 16.6321428 18.7740189 15.7728413 23.4693045 11.3206157 25.9411644 13.637089 31.1059383 18.1230901 31.1059383L32.7746407 31.1059383C35.477997 31.1059383 37.7624493 29.2110739 37.7624493 26.9542284 37.7624493 24.6474544 35.3148218 22.5552921 32.2057208 22.8270799" opacity=".08"/>
    <polyline fill="#0A5A7E" points="72.255 141.992 72.255 70.498 124.105 70.498 124.105 82.554 131.94 82.554 131.94 60.47 135.224 60.47 149.762 40.41 164.755 60.47 169.226 60.47 169.226 91.932 179.524 91.932 179.524 20.978 213.34 20.978 213.34 64.416 222.996 64.416 222.996 56.106 249.688 56.106 249.688 64.223 272.444 64.223 272.444 27.86 278.618 27.86 294.458 5 307.359 27.86 314.317 27.86 314.317 63.733 332.669 63.733 332.669 53.966 346.655 53.966 346.655 63.19 350.9 63.19 350.9 37.036 374.037 37.036 374.037 62.372 394.421 62.372 394.421 23.397 421.1 23.397 421.1 141.992 74.001 141.992"/>
    <g transform="matrix(-1 0 0 1 175.837 17.184)">
      <g transform="translate(93.861 13.244)">
        <polygon fill="#2F81A8" points="26.841 33.388 26.016 112.312 30.682 112.312 29.857 33.388"/>
        <g transform="translate(0 .037)">
          <path fill="#2F81A8" d="M31.6626096,26.8311103 C32.9314261,27.9948676 32.9314261,31.8604887 31.6626096,33.024305 C30.393986,34.1880623 26.1796911,34.1880033 24.9110032,33.024305 C23.642444,31.8606067 23.642444,27.9949266 24.9110032,26.8311103 C26.1797554,25.667353 30.393986,25.667353 31.6626096,26.8311103 Z"/>
          <g transform="translate(26.885)">
            <mask id="b" fill="white">
              <use xlink:href="#a"/>
            </mask>
            <polygon fill="#2F81A8" points="1.92 26.671 .884 26.671 .032 21.52 1.049 .022 1.507 .022 2.332 21.595" mask="url(#b)"/>
          </g>
          <polygon fill="#2F81A8" points="27.604 27.016 28.97 27.016 28.97 26.543 27.604 26.543"/>
          <polygon fill="#2F81A8" points="29.108 26.811 27.465 26.811 27.342 27.679 29.232 27.679"/>
          <polygon fill="#2F81A8" points="24.953 31.145 25.471 31.968 21.034 35.219 .229 45.161 0 44.797 19.955 33.354"/>
          <polygon fill="#2F81A8" points="25.192 30.843 25.875 31.929 25.433 32.162 24.75 31.077"/>
          <polygon fill="#2F81A8" points="24.933 30.834 25.755 32.139 26.637 31.803 25.692 30.301"/>
          <polygon fill="#2F81A8" points="31.103 31.968 31.621 31.145 36.91 33.044 56.698 44.6 56.468 44.964 35.688 34.833"/>
          <polygon fill="#2F81A8" points="30.699 31.929 31.382 30.843 31.824 31.077 31.14 32.162"/>
          <polygon fill="#2F81A8" points="30.819 32.139 31.64 30.834 30.882 30.301 29.937 31.803"/>
          <path fill="#2F81A8" d="M31.1929732,29.9276958 C31.1929732,31.3999594 29.8918686,32.5934523 28.2868578,32.5934523 C26.6818471,32.5934523 25.3807425,31.3999594 25.3807425,29.9276958 C25.3807425,28.4554322 26.6818471,27.2619394 28.2868578,27.2619394 C29.8918686,27.2619394 31.1929732,28.4554322 31.1929732,29.9276958"/>
          <path fill="#2F81A8" d="M30.4815689,29.9276958 C30.4815689,31.0395929 29.4989709,31.9408632 28.2868836,31.9408632 C27.0747319,31.9408632 26.0921983,31.0395929 26.0921983,29.9276958 C26.0921983,28.8158578 27.0747319,27.9145285 28.2868836,27.9145285 C29.4989709,27.9145285 30.4815689,28.8158578 30.4815689,29.9276958"/>
        </g>
      </g>
      <polygon fill="#2F81A8" points="25.555 21.072 24.477 109.691 29.207 109.583 28.128 20.964"/>
      <g transform="translate(0 .008)">
        <path fill="#2F81A8" d="M29.6440763,15.5379216 C30.7200153,16.5244758 30.7200153,19.8015364 29.6440763,20.7880406 C28.5683008,21.7746949 24.9946364,21.7745949 23.9188064,20.7880406 C22.8430855,19.8015864 22.8430855,16.5245259 23.9188064,15.5379216 C24.9946909,14.5513674 28.5683008,14.5513674 29.6440763,15.5379216"/>
        <polygon fill="#2F81A8" points="27.137 20.934 27.985 20.726 29.915 24.772 34.226 42.58 33.851 42.672 28.013 25.173"/>
        <polygon fill="#2F81A8" points="26.92 20.689 28.039 20.414 28.151 20.797 27.032 21.072"/>
        <polygon fill="#2F81A8" points="26.854 20.881 28.2 20.55 28.094 19.814 26.545 20.194"/>
        <g transform="translate(28.58)">
          <mask id="d" fill="white">
            <use xlink:href="#c"/>
          </mask>
          <polygon fill="#2F81A8" points=".642 16.495 .02 15.926 2.877 12.37 17.539 .042 17.814 .294 4.206 13.68" mask="url(#d)"/>
        </g>
        <polygon fill="#2F81A8" points="29.098 16.79 28.278 16.039 28.584 15.759 29.403 16.51"/>
        <polygon fill="#2F81A8" points="29.312 16.746 28.326 15.843 27.684 16.296 28.818 17.335"/>
        <polygon fill="#2F81A8" points="23.987 17.059 23.759 17.837 18.973 17.347 0 11.867 .101 11.522 19.546 15.636"/>
        <polygon fill="#2F81A8" points="24.327 17.01 24.027 18.036 23.61 17.933 23.91 16.907"/>
        <polygon fill="#2F81A8" points="24.179 16.862 23.818 18.096 24.566 18.38 24.981 16.96"/>
        <path fill="#2F81A8" d="M24.4011709,18.7478715 C24.0488877,17.5422997 24.8290486,16.3031676 26.1436064,15.980168 C27.4583279,15.6571184 28.8096467,16.372489 29.1619299,17.5780608 C29.5142131,18.7836325 28.7339432,20.0228147 27.4192762,20.3458643 C26.1046638,20.6688639 24.7534541,19.9533933 24.4011709,18.7478715"/>
        <path fill="#2F81A8" d="M24.9838154,18.6047124 C24.7178167,17.6942316 25.3069734,16.758443 26.2997913,16.5144678 C27.2926637,16.2705425 28.3131341,16.8108091 28.5791873,17.7212399 C28.8452405,18.6316707 28.2560292,19.5675093 27.2631568,19.8114845 C26.2703389,20.0554597 25.2498685,19.5150931 24.9838154,18.6047124"/>
      </g>
      <g fill="#2F81A8" transform="translate(52.603 11.352)">
        <polygon points="25.508 21.041 24.62 97.389 28.948 97.389 28.06 21.041"/>
        <g transform="translate(0 .002)">
          <path d="M29.5888736,15.5200396 C30.6628053,16.5051379 30.6628053,19.7773122 29.5888736,20.7624105 C28.5151052,21.7475587 24.9480536,21.7475088 23.8742308,20.7624105 C22.8005168,19.7774121 22.8005168,16.5051879 23.8742308,15.5200396 C24.9481625,14.5349414 28.5151052,14.5349414 29.5888736,15.5200396"/>
          <polygon points="23.942 17.039 23.715 17.816 18.938 17.326 0 11.854 .1 11.51 19.509 15.618"/>
          <polygon points="24.282 16.99 23.982 18.015 23.566 17.912 23.865 16.887"/>
          <polygon points="24.133 16.842 23.774 18.074 24.521 18.358 24.935 16.94"/>
          <polygon points="27.086 20.908 27.933 20.7 29.859 24.741 34.162 42.522 33.787 42.614 27.961 25.141"/>
          <polygon points="26.87 20.663 27.987 20.389 28.098 20.771 26.981 21.046"/>
          <polygon points="26.804 20.855 28.148 20.525 28.041 19.79 26.496 20.17"/>
          <polygon points=".64 16.476 .02 15.907 2.871 12.357 17.506 .047 17.781 .299 4.198 13.665" transform="translate(28.526)"/>
          <polygon points="29.043 16.77 28.226 16.021 28.531 15.741 29.349 16.491"/>
          <polygon points="29.257 16.727 28.274 15.825 27.633 16.277 28.764 17.314"/>
          <path d="M27.3682243,15.9616383 C28.6804385,16.2842112 29.4591438,17.5214647 29.1075179,18.7251574 C28.7558919,19.9289999 27.4072031,20.6434646 26.0949345,20.3208917 C24.7827747,19.9983688 24.0040694,18.7609655 24.3556954,17.557123 C24.7072668,16.3534303 26.0560101,15.6391154 27.3682243,15.9616383"/>
          <path d="M27.212456,16.4951446 C28.2034217,16.7387598 28.7914247,17.6731673 28.5259224,18.5822045 C28.26042,19.4913915 27.241799,20.0308608 26.2508334,19.7872956 C25.2598678,19.5436804 24.6717558,18.6092728 24.9372582,17.7000858 C25.202815,16.7910486 26.2214904,16.2515794 27.212456,16.4951446"/>
        </g>
      </g>
    </g>
    <path fill="#106890" d="M733,70.7104606 C733,70.7104606 269.134935,38.4098309 0.29296875,142.975238 L732.808621,142.975238 L733,70.7104606 Z"/>
    <path fill="#1375A2" d="M204.186146,143 C204.186146,143 303.643764,137.992942 281.704584,120.468658 C259.765403,102.944374 304.521331,94.1822322 304.521331,94.1822322 C304.521331,94.1822322 363.610858,80.914324 380.284635,84.6694081 C380.284635,84.6694081 320.025019,89.9266933 300.133495,103.195438 C285.214852,111.456289 309.786735,115.4616 309.786735,115.4616 C309.786735,115.4616 338.673907,121.533171 334.286071,142.812539 L204.186146,143 Z"/>
  </g>
</svg>
