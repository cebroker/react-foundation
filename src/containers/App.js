import { connect } from 'react-redux';
import { actions, selectors } from '../store/loggedInUser';
import { App } from '../components';

function mapStateToProps(state) {
  const loggedInUser = selectors(state);
  return {
    user: loggedInUser.getUser(),
    userFetchStatus: loggedInUser.getFetchStatus()
  };
}

export default connect(mapStateToProps, {
  fetchLoggedInUser: actions.fetch
})(App);
