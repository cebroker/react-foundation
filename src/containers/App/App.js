import React, { Component } from 'react';
import { connect } from 'react-redux';

import { actions, selectors } from '../../store/loggedInUser';
import { App } from '../../components';

class AppContainer extends Component {
  componentDidMount() {
    const { userFetchStatus, fetchLoggedInUser } = this.props;
    if (userFetchStatus === 'notLoaded') {
      fetchLoggedInUser();
    }
  }
  render() {
    return <App user={this.props.user} />;
  }
}

function mapStateToProps(state) {
  const loggedInUser = selectors(state);
  return {
    user: loggedInUser.getUser(),
    userFetchStatus: loggedInUser.getFetchStatus()
  };
}

export default connect(mapStateToProps, {
  fetchLoggedInUser: actions.fetch
})(AppContainer);
