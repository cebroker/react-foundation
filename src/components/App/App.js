import React, { Component } from 'react';
import {
  Nav,
  NavItem,
  RedIndicator,
  SearchForm,
  Avatar,
  NavDropdown,
  MenuItem
} from '@isfco/evercheck-ui';
import AppBar from '../AppBar';
import ceBrokerLogo from './ce-broker-logo.svg';
import './App.css';
import { LinkContainer } from 'react-router-bootstrap';

class App extends Component {
  render() {
    return (
      <div className="App">
        <AppBar>
          <AppBar.Header>
            <AppBar.Brand target="/" className="App-brand">
              <img src={ceBrokerLogo} alt="[My APP]" className="App-logo" />
            </AppBar.Brand>
          </AppBar.Header>
          <Nav>
            <LinkContainer to="#">
              <NavItem eventKey={1}>
                My records
              </NavItem>
            </LinkContainer>
            <LinkContainer to="#">
              <NavItem eventKey={2}>
                Browse courses
              </NavItem>
            </LinkContainer>
          </Nav>
          <div className="pull-right">
            <SearchForm
              placeholder="Search for a course provider or subject"
              inputId="query"
              onSubmit={e => e.preventDefault()}
            />
            <Nav pullRight>
              <LinkContainer to="#" className="upgrade-link">
                <NavItem eventKey={1}>
                  UPGRADE NOW
                </NavItem>
              </LinkContainer>
              <LinkContainer to="#" className="nav-item-icon">
                <NavItem eventKey={2}>
                  <RedIndicator>
                    <i className="material-icons" style={{ fontSize: '26px' }}>mail</i>
                  </RedIndicator>
                </NavItem>
              </LinkContainer>
              <NavDropdown
                className="avatar-dropdown"
                eventKey={3}
                title={<Avatar title="John Doe" />}
                id="test-nav-dropdown"
                pullRight
                noCaret
              >
                <MenuItem
                  header
                  style={{
                    display: 'flex',
                    alignItems: 'center',
                    paddingTop: '11px',
                    paddingBottom: '10px',
                    minWidth: '217px'
                  }}
                >
                  <Avatar
                    title="John Doe"
                    size="lg"
                    style={{ display: 'inline-block', marginRight: '20px' }}
                  />
                  <div style={{ display: 'inline-block' }}>
                    <p className="username">John Doe</p>
                    <p className="subtitle">ADMIN</p>
                  </div>
                </MenuItem>
                <MenuItem divider />
                <LinkContainer to="#">
                  <MenuItem icon="settings" eventKey={3.1}>Account settings</MenuItem>
                </LinkContainer>
                <LinkContainer to="#">
                  <MenuItem icon="supervisor_account" eventKey={3.2}>Users</MenuItem>
                </LinkContainer>
                <MenuItem divider />
                <LinkContainer to="#">
                  <MenuItem icon="exit_to_app" eventKey={3.3}>Sign out</MenuItem>
                </LinkContainer>
              </NavDropdown>
            </Nav>
          </div>
        </AppBar>
      </div>
    );
  }
}

export default App;
