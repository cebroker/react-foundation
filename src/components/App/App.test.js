import React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import App from './App';

const user = { firstName: 'Jhon', lastName: 'Doe' };

it('renders without crashing', () => {
  shallow(<App user/>);
});

it('renders correctly - snapshot', () => {
  const tree = renderer.create(<MemoryRouter>
      <App user/>
    </MemoryRouter>);
  expect(tree).to.matchSnapshot();
});
