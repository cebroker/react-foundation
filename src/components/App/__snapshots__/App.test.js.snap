// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly - snapshot 1`] = `
<div
  className="App"
>
  <nav
    className="AppBar appbar navbar navbar-inverse"
  >
    <div
      className="container"
    >
      <div
        className="navbar-header"
      >
        <a
          className="navbar-brand App-brand"
          href="/"
          onClick={[Function]}
        >
          <img
            alt="[My APP]"
            className="App-logo"
            src="ce-broker-logo.svg"
          />
        </a>
      </div>
      <ul
        className="nav"
        role={null}
      >
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="#"
            onClick={[Function]}
            role="button"
          >
            My records
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="#"
            onClick={[Function]}
            role="button"
          >
            Browse courses
          </a>
        </li>
      </ul>
      <div
        className="pull-right"
      >
        <form
          className="search-form"
          id={undefined}
          onClick={[Function]}
          onSubmit={[Function]}
          style={undefined}
        >
          <i
            className="material-icons"
          >
            search
          </i>
          <label
            className="sr-only"
            htmlFor="query"
          >
            Search for a course provider or subject
          </label>
          <input
            autoCapitalize="off"
            autoComplete="off"
            autoCorrect="off"
            id="query"
            name="query"
            onChange={[Function]}
            placeholder="Search for a course provider or subject"
            type="search"
            value=""
          />
        </form>
        <ul
          className="nav pull-right"
          role={null}
        >
          <li
            className="upgrade-link"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              onClick={[Function]}
              role="button"
            >
              UPGRADE NOW
            </a>
          </li>
          <li
            className="nav-item-icon"
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              onClick={[Function]}
              role="button"
            >
              <div
                className="red-indicator active"
              >
                <i
                  className="material-icons"
                  style={
                    Object {
                      "fontSize": "26px",
                    }
                  }
                >
                  mail
                </i>
              </div>
            </a>
          </li>
          <li
            className="avatar-dropdown dropdown"
            style={undefined}
          >
            <a
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle"
              href="#"
              id="test-nav-dropdown"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              <div
                className="avatar avatar-xs"
                style={
                  Object {
                    "backgroundColor": "#009688",
                  }
                }
              >
                uu
              </div>
            </a>
            <ul
              aria-labelledby="test-nav-dropdown"
              className="dropdown-menu dropdown-menu-right"
              role="menu"
            >
              <li
                className="dropdown-header"
                onKeyDown={[Function]}
                role="heading"
                style={
                  Object {
                    "alignItems": "center",
                    "display": "flex",
                    "minWidth": "217px",
                    "paddingBottom": "10px",
                    "paddingTop": "11px",
                  }
                }
              >
                <div
                  className="avatar avatar-lg"
                  style={
                    Object {
                      "backgroundColor": "#009688",
                      "display": "inline-block",
                      "marginRight": "20px",
                    }
                  }
                >
                  uu
                </div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                    }
                  }
                >
                  <p
                    className="username"
                  >
                    undefined undefined
                  </p>
                  <p
                    className="subtitle"
                  >
                    ADMIN
                  </p>
                </div>
              </li>
              <li
                className="divider"
                onKeyDown={[Function]}
                role="separator"
                style={undefined}
              />
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="menuitem"
                  tabIndex="-1"
                >
                  <i
                    className="ec-icon material-icons"
                  >
                    settings
                  </i>
                  Account settings
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="menuitem"
                  tabIndex="-1"
                >
                  <i
                    className="ec-icon material-icons"
                  >
                    supervisor_account
                  </i>
                  Users
                </a>
              </li>
              <li
                className="divider"
                onKeyDown={[Function]}
                role="separator"
                style={undefined}
              />
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="menuitem"
                  tabIndex="-1"
                >
                  <i
                    className="ec-icon material-icons"
                  >
                    exit_to_app
                  </i>
                  Sign out
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <div
    className="ec-page-header"
  >
    <div
      className="container"
    >
      <img
        alt="illustration of buildings and windmills"
        className="page-header-illustration"
        src="page-header-illustration.svg"
      />
      <h2
        className="text-left"
        style={
          Object {
            "marginBottom": "25px",
            "marginTop": "29px",
          }
        }
      >
        Good afternoon, 
         !
      </h2>
      <ul
        className="page-header-nav nav nav-pills"
        role={null}
      >
        <li
          className="active"
          role="presentation"
          style={undefined}
        >
          <a
            href="#"
            onClick={[Function]}
            role="button"
          >
            OVERVIEW
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="#"
            onClick={[Function]}
            role="button"
          >
            COURSE HISTORY
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="#"
            onClick={[Function]}
            role="button"
          >
            REPORTED EXEMPTIONS
          </a>
        </li>
        <li
          className="dropdown"
          style={undefined}
        >
          <a
            aria-expanded={false}
            aria-haspopup={true}
            className="dropdown-toggle"
            href="#"
            id="tab-dropdown"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
          >
            <span>
              MORE
              <i
                className="material-icons"
              >
                arrow_drop_down
              </i>
            </span>
          </a>
          <ul
            aria-labelledby="tab-dropdown"
            className="dropdown-menu"
            role="menu"
          >
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="menuitem"
                tabIndex="-1"
              >
                <i
                  className="ec-icon material-icons"
                >
                  settings
                </i>
                Action 1
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="menuitem"
                tabIndex="-1"
              >
                <i
                  className="ec-icon material-icons"
                >
                  supervisor_account
                </i>
                Action 2
              </a>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="container"
    style={
      Object {
        "paddingTop": "20px",
      }
    }
  >
    <h2
      className="text-left"
    >
      Overview
    </h2>
  </div>
</div>
`;
